css-selectors/selectorsList.js

```diff
@@ -0,0 +1,10 @@
+const titleSelector = 'title#hgvdhet_';
+const headerSelector = 'header.header h1[data-cy="header"]';
+const websiteLinkSelector = 'a[href="https://www.cypress.io/"]';
+const descriptionSelector = 'p.description';
+const button1Selector = 'button#button1';
+const button2Selector = 'button#button2';
+const button3Selector = 'button#button3';
```

Comment:
unnecessary to write tags with attributes in this particular case. Our id is unique, to you are able to make these selectors shorter:
```suggestion
const button1Selector = '#button1';
const button2Selector = '#button2';
const button3Selector = '#button3';
```